src/scenic/core/pruning.py

```diff
@@ -135,8 +146,6 @@ def matchPolygonalField(heading, position):
 
 
 ### Pruning procedures
-
-
```

Comment:
I would like to leave at least 1 blank line after these headings, since they aren't comments referring to a single function but blocking out a whole section of the file.

Reply:
Ah good point, I'll re-add the spaces.